cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1459,7 +1459,7 @@ def test_json_responses(self):
         self.validate_course_xblock_info(json_response, course_outline=True)
 
     @ddt.data(
-        (ModuleStoreEnum.Type.split, 5, 5),
+        (ModuleStoreEnum.Type.split, 4, 4),
```

Comment:
@chrisndodge  @cpennington I could not understand the this behavior , After changing the scope of `is_entrance_exam` field, Split store expecting 4 calls instead of 5 now ? I updated its value But could not understand the inner logic in case of Split. Can you guys share your thoughts on this behavior ?